_drafts/oryx/technology.md

```diff
@@ -0,0 +1,66 @@
+# Oryx technologies
+
+Oryx is a frontend framework that is designed to help developers build web applications using modern technologies.
+When possible and appropriate, Oryx adheres to standards for both technologies and tools.
+Oryx strives to minimize the use of build tools and instead utilize the native capabilities of the web, as much as possible.
+Oryx aims to use JavaScript during development, but tries avoid the need to run JavaScript in the browser. Wherever possible, HTML and CSS are preferred to render a fast loading experience cross different screen sizes.
+
+We will examine both the deployed technology stack and the technologies utilized for development in order to gain a more comprehensive understanding of the technologies.
+
+## Deployed stack:
```

Comment:
I do like the fact that we articulate deployed tech vs developer tech. But it feels overcomplex and like repetition when you first read it. What about this:
Have a single section (here at the top), describing about how we embrace standard web platform. We do not need to go deep and fine grained. But we do talk a little about web components, avoiding JS in te browser, etc. But we keep it grips, and then have the list of Technologies to build Oryx applications.